.github/workflows/push-ds-image.yaml

```diff
@@ -0,0 +1,52 @@
+name: image-publisher
+
+on:
+  workflow_dispatch:
+    inputs:
+      REGISTRY_NAME:
+        required: true
+        default: quay.io/ocs-dev
+      IMAGE_TAGS:
```

Comment:
`IMAGE_TAGS` to `IMAGE_TAG` and also if this is required we dont need default isnt it?